### ISSUE TITLE
The color indication for completed page in Clarity Wizard does not have sufficient contrast

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#4290]
## The colored marker in Clarity Wizard TOC marking a successfully completed page does not have enough contrast with its background

Regarding the WCAG recommendations foreground and background colors should have at least 3:1 . The colored marker has lower contrast than the recommended.

## How to reproduce
Open any Clarity wizard and complete the first page.
When you go to the next one, there is a green left border on TOC item which represents the completed page.
The green is not with enough contrast its 2.48:1 and should be at least 3:1

## Expected behavior
The green marker should have at least 3:1 contrast with its background

## Versions

**App**

* Angular: 7.2.14
* Clarity: 1.3.2

**Device:**

* Type: [MacBook]
* OS: [OSX]
* Browser [Chrome]
* Version [7.9]

## Additional notes

_Add any other notes about the problem here._

**Comment by gnomeontherun** [comment]
Need to determine backportability.

**Comment by gnomeontherun** [comment]
An application can also override the value as necessary for a workaround in their CSS.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
